src/daemon/common_functions.sh

```diff
@@ -519,16 +519,28 @@ function ami_privileged {
 
 # Detect how much ram is available
 function get_available_ram {
-  limit_in_bytes="/sys/fs/cgroup/memory/memory.limit_in_bytes"
-  memory_limit=$(cat $limit_in_bytes)
+  memory_limit_in_bytes_file=""
+  memory_current_usage_in_bytes_file=""
+
+  if grep -q cgroup2 /proc/filesystems; then
+    # cgroups v2
+    memory_limit_in_bytes_file="/sys/fs/cgroup/memory.max"
+    memory_current_usage_in_bytes_file="/sys/fs/cgroup/memory.current"
```

Comment:
@KannarFr 

```console
[root@7f9056067200 /]# grep cgroup2 /proc/filesystems
nodev	cgroup2
[root@7f9056067200 /]# ls /sys/fs/cgroup/memory.max
ls: cannot access '/sys/fs/cgroup/memory.max': No such file or directory
[root@7f9056067200 /]# ls /sys/fs/cgroup/memory.current
ls: cannot access '/sys/fs/cgroup/memory.current': No such file or directory
[root@7f9056067200 /]#
```

from the documentation [1] :
>  memory.usage_in_bytes		 # show current usage for memory
 memory.limit_in_bytes		 # set/show limit of memory usage


did you mean `/sys/fs/cgroup/memory/memory.limit_in_bytes`  `/sys/fs/cgroup/memory/memory.usage_in_bytes` ?

Reply:
@KannarFr forget about this comment, I think I got confused.